setup.py

```diff
@@ -0,0 +1,57 @@
+import pathlib
+
+from skbuild import setup
+
+def exclude_static_libraries(cmake_manifest):
+    return list(filter(lambda name: not (name.endswith(".a") and not name.endswith(".lib")), cmake_manifest))
+
+here = pathlib.Path(__file__).parent.resolve()
+long_description = (here / "README.md").read_text(encoding="utf-8")
+
+setup(
+    name="f3d",
+    description="F3D, a fast and minimalist 3D viewer",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://f3d.app",
+    project_urls={
+        "Source Code": "https://github.com/f3d-app/f3d",
+        "Bug Tracker": "https://github.com/f3d-app/f3d/issues",
+        "Documentation": "https://f3d.app",
+    },
+    author="Michael Migliore and Mathieu Westphal",
+    author_email="mcmigliore+pip@gmail.com",
+    license="BSD 3-Clause",
+    license_files=["LICENSE.md"],
+    classifiers=[  # TODO
+        "License :: OSI Approved :: BSD License",
+        "Operating System :: Microsoft :: Windows",
+        "Operating System :: POSIX",
+        "Operating System :: MacOS",
+        "Programming Language :: C++",
+        "Programming Language :: Python :: 3",
+    ],
+    keywords="vtk;animations;fbx;step;stl;dcm;ex2;gml;obj;3ds;gltf;pbr;raytracing;scivis;volume;rendering",
+    python_requires=">=3.7",
+    packages=["f3d"],
+    package_dir={
+        "f3d": "python/packaging",
+    },
+    install_requires=[],
+    entry_points={"console_scripts": ["f3d=f3d.__main__:main"]},
+    cmake_source_dir=".",
+    cmake_install_dir="python/packaging",
+    cmake_args=[  # TODO: enable/disable modules with env variables?
+        "-DF3D_BINDINGS_PYTHON:BOOL=ON",
+        "-DBUILD_SHARED_LIBS:BOOL=OFF",
+        "-DF3D_PLUGINS_STATIC_BUILD:BOOL=ON"
+        "-DF3D_PLUGIN_BUILD_ALEMBIC:BOOL=ON",
+        "-DF3D_PLUGIN_BUILD_ASSIMP:BOOL=ON",
+        "-DF3D_PLUGIN_BUILD_DRACO:BOOL=ON",
+        "-DF3D_PLUGIN_BUILD_EXODUS:BOOL=ON",
+        "-DF3D_PLUGIN_BUILD_OCCT:BOOL=ON",
+        "-DF3D_MODULE_EXR:BOOL=ON",
```

Comment:
Depending on who the temporary (as all of that would eventually be solved with a proper PyPI build) "target audience" is you may want to scale it down to stuff that we can expect to be obtainable without having to be compiled "by hand"?

For example on Fedora I can get the Assimp deps easy enough with the package manager, but the default VTK that doesn't have EXR.  Not sure about the other plugins.

Reply:
I agree, I think we should remove all optional dependencies and allow activation somehow (env var?)

Reply:
env var suggestion in separate comment

Reply:
Thanks